packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -43,29 +56,46 @@ export class DemoComponent implements OnInit {
     if (Object.keys(translations).length == 0) {
       return false;
     } else {
-      const sentence_nodes = doc.querySelectorAll("s:not(.sentence__translation)")
+      const sentence_nodes = doc.querySelectorAll(
+        "s:not(.sentence__translation)"
+      );
       // represents all translation nodes that have already been added
-      const translation_node_ids = new Set(Array.from(doc.querySelectorAll(".editable__translation")).map((t_node) => t_node.id))
+      const translation_node_ids = new Set(
+        Array.from(doc.querySelectorAll(".editable__translation")).map(
+          (t_node) => t_node.id
+        )
+      );
       sentence_nodes.forEach((sentence: Element) => {
         // Add a translation
-        if (sentence.id in translations && !translation_node_ids.has(sentence.id)) {
+        if (
+          sentence.id in translations &&
+          !translation_node_ids.has(sentence.id)
+        ) {
           // No namespaces!! NO! NO! NO!
```

Comment:
hahah I like this comment.

Reply:
thanks :) note that it really seems to be some undefined browser behaviour that's causing this problem, I would be interested to see if it happens on Safari or Firefox too